docs/reST/tut/tom_games6.rst

```diff
@@ -319,7 +319,7 @@ like this::
           pygame.display.flip()
 
 
-  if __name__ == '__main__': main()
+  if __name__ == "__main__": main()
```

Comment:
How about moving the main() under the if statement, that seems more conventional.